### DISCUSSION TITLE
How can I add a table with multiple columns and rows, inside a content?

### DISCUSSION THREAD

**Discussion by Sarvom** [fastn-stack/fastn#908] [Q&A]
How can I add a table with multiple columns and rows, inside a content?  Should I use border width property?

**Comment by gsalunke** [comment]
@Sarvom 
as of now `table` isn't implemented into `FTD` but using `row` and `columns` you can create something like:

Invocation link: https://github.com/fastn-community/site-doc/blob/main/component-doc.ftd#L89
Code table header: https://github.com/fastn-community/site-doc/blob/edcc2ca301d542196ef14007d2025d52b9948e7c/index.ftd#L1058
Code table body: https://github.com/fastn-community/site-doc/blob/edcc2ca301d542196ef14007d2025d52b9948e7c/index.ftd#L1108
Demo:
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/68585007/236768287-1a6a69f9-a8d1-4ad3-a86a-66448b9d7d13.png">

URL: https://fastn-community.github.io/site-doc/component-doc/
Click on Documentation tab - and scroll to bottom.
